.github/workflows/vc-command.yml

```diff
@@ -53,7 +53,7 @@ jobs:
               -vvvv --os-version $OS_VERSION \
                 add_changelog_entry \
                 --user $user \
```

Comment:
and here?

Reply:
Usernames should not contain spaces so it shouldn't be necessary :crossed_fingers:

Reply:
quoting handles more than just spaces (for example usernames like '-miska-' are valid, but need to be quoted in shell)